packages/loaders/src/styled-elements.js

```diff
@@ -23,42 +23,62 @@ StyledCircle.propTypes = {
   transform: PropTypes.string
 };
 
-const COMPONENT_ID = 'loaders.dots';
+export const SpinnerCircle = styled.circle.attrs({
+  cx: 40,
+  cy: 40,
+  r: 34,
+  fill: 'none',
+  stroke: 'currentColor',
+  strokeLinecap: 'round',
+  strokeWidth: ({ strokeWidthValue }) => strokeWidthValue,
+  strokeDasharray: ({ dasharrayValue }) => `${dasharrayValue} 250`,
+  transform: props => props.transform
+})`
+  transform-origin: center center;
+`;
+
+SpinnerCircle.propTypes = {
+  dasharrayValue: PropTypes.string,
+  strokeWidthValue: PropTypes.string,
+  transform: PropTypes.string
+};
 
 const StyledSvg = styled.svg.attrs({
-  'data-garden-id': COMPONENT_ID,
+  'data-garden-id': ({ dataGardenId }) => dataGardenId,
   'data-garden-version': PACKAGE_VERSION,
   xmlns: 'http://www.w3.org/2000/svg',
-  width: 80,
-  height: 72,
+  width: ({ width }) => width,
+  height: ({ height }) => height,
   focusable: 'false',
-  viewBox: '0 0 80 72',
+  viewBox: ({ width, height }) => `0 0 ${width} ${height}`,
   role: 'progressbar'
 })`
   width: 1em;
   height: 0.9em;
   color: ${props => props.color || 'inherit'};
   font-size: ${props => props.fontSize || 'inherit'};
 
-  ${props => retrieveTheme(COMPONENT_ID, props)};
+  ${props => retrieveTheme(props.dataGardenId, props)};
 `;
 
 StyledSvg.propTypes = {
   color: PropTypes.string,
   fontSize: PropTypes.any
 };
 
-export const StyledSVG = ({ children, fontSize, ...other }) => {
+export const StyledSVG = ({ children, fontSize, width, height, ...other }) => {
   return (
-    <StyledSvg fontSize={fontSize} {...other}>
+    <StyledSvg fontSize={fontSize} width={width} height={height} {...other}>
       <g fill="currentColor">{children}</g>
```

Comment:
This `<g>` element should not be applied to the spinner's SVG – it's only applicable to the grouping of Dots.